### PR TITLE
Use lower max degree by default and let user change it

### PR DESCRIPTION
**Pull request by leonardoalt**
Depends on https://github.com/powdr-labs/powdr/pull/1946

### CODE REVIEW

File: powdr/src/lib.rs [hunk 1]
```diff
@@ -29,10 +29,35 @@ pub struct Session {
 const DEFAULT_PKEY: &str = "pkey.bin";
 const DEFAULT_VKEY: &str = "vkey.bin";
 
+// Minimum and maximum log of number of rows for the RISCV machine.
+const DEFAULT_LOG_MIN_DEGREE: u32 = 5;
```
Comment (pacheco) [inline powdr/src/lib.rs:33]: `MIN/MAX_DEGREE_LOG` to be consistent with the cmd line option?

Comment (pacheco) [reply powdr/src/lib.rs:33]: same for the related names of args and variables

**Comment by pacheco** [comment]
what's the usage for the user setting the MIN_DEGREE?

**Comment by leonardoalt** [comment]
> what's the usage for the user setting the MIN_DEGREE?

not much, that's why I didn't expose it to the end user

**Comment by leonardoalt** [comment]
updated, waiting for the other PR to be merged

**Comment by leonardoalt** [comment]
this is ready now

**Review by pacheco — APPROVED** [review]

**Pull request merged**